README.md

```diff
@@ -100,7 +100,7 @@ make build
 
 5. Run the Application Set Controller from the CLI:
 ```
-NAMESPACE=argocd ./dist/argocd-applicationset --metrics-addr=":18081" --probe-addr=":18082"
+NAMESPACE=argocd ./dist/argocd-applicationset --metrics-addr=":18081" --probe-addr=":18082" --argocd-repo-server=localhost:8081 --debug  --namespace=argocd
```

Comment:
By adding `--namespaces=argocd`, we can elminite `NAMESPACE=argocd`?

Reply:
Yah, I'm going to clean this up as part of https://github.com/argoproj-labs/applicationset/issues/70 , but until that bug is fixed both are needed.